ZenPacks/zenoss/OpenStackInfrastructure/modeler/plugins/zenoss/OpenStackInfrastructure.py

```diff
@@ -504,6 +562,10 @@ def process(self, device, results, unused):
         hosts = []
         for host_id in hostmap:
             data = hostmap[host_id]
+
+            if not isValidHostname(data.get('hostname', host_id)):
```

Comment:
What is it if it's not a hostname?   Should it at least log a message about this invalid data that it's ignoring?